src/_premake_init.lua

```diff
@@ -1089,11 +1089,32 @@
 		tokens = true,
 	}
 
+	api.deprecateValue("flags", "MFC", 'Use `mfc` instead.',
+	function(value)
+		mfc("On")
+	end,
+	function(value)
+		mfc("Off")
+	end)
+
 	api.deprecateField("sysincludedirs", 'Use `externalincludedirs` instead.',
 	function(value)
 		externalincludedirs(value)
 	end)
 
+	api.register {
```

Comment:
modules/vstudio/_preload.lua seems a better place for the registration

Reply:
Good call. This does appear to only affect VS.